changelog.d/pa-3070.fixed

```diff
@@ -0,0 +1,6 @@
+Implemented key path expression parsing in Swift. The following example should
+now be correctly matched by the \$X.isActive pattern:
```

Comment:
and $X will bind to which content in this example?

Reply:
Done.